docs/pages/eas-update/github-actions.mdx

```diff
@@ -161,3 +161,41 @@ Your GitHub Action should be set up now. Whenever a developer creates a pull req
 > Some repositories or organizations might need to explicitly enable GitHub Workflows and allow third-party Actions.
 
 </Step>
+
+## Using Bun instead of Yarn
+
+If you're using Bun as your package manager instead of Yarn, you'll need to make some modifications to the provided workflow. Below are the updated steps for both publishing updates on push and previews on pull requests:
+
+Publish updates on push using Bun:
+
+<Step label="1">
+
+Replace the Node setup steps in update.yml:
+
+```- name: Setup Node
+  uses: actions/setup-node@v3
+  with:
+    node-version: 18.x
+    cache: yarn
+```
+with
+```- name: Setup Bun
+  uses: oven-sh/setup-bun@v1
+  with:
+    bun-version: latest
+```
+</Step>
+
+<Step label="2">
+
+Install Dependencies with Bun:
+
+Replace the dependency installation step:
+
+```- name: Install dependencies
+  run: yarn install
+```
+with
+```- name: Install dependencies
+  run: bun install
+```
```

Comment:
````suggestion
```yaml update.yml
- name: Install dependencies
  run: bun install
```
````

Format the code snippet to use the language syntax and the file name. The change suggested here will end up as following:

![CleanShot 2023-10-30 at 12 57 14@2x](https://github.com/expo/expo/assets/10234615/52d4f8c4-c104-4377-82c3-62921e5eed31)

Which is how we represent most code snippets in Expo documentation.